python/lsst/pipe/analysis/analysis.py

```diff
@@ -1094,6 +1098,91 @@ def plotQuiver(self, catalog, description, plotInfoDict, areaDict, log, cmap=plt
 
         yield Struct(fig=fig, description=description, stats=stats, statsHigh=None, dpi=150, style="quiver")
 
+    def plotRhoStatistics(self, description, plotInfoDict, log, treecorrParams,
+                          stats, butler=None, camera=None,
+                          ccdList=None, tractInfo=None, patchList=None, hscRun=None, matchRadius=None,
+                          zpLabel=None, forcedStr=None, postFix="", flagsCat=None, uberCalLabel=None):
```

Comment:
Please can you get rid of the kwargs that aren't used? butler, camera, ccdList, tractInfo, patchList and hscRun should all either be gone or be in plotInfoDict.